### PR TITLE
EVG-2722 don't allow selecting execution tasks in patches from CLI

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: model/project.go [hunk 1]
```diff
@@ -750,6 +750,9 @@ func (p *Project) FindTaskForVariant(task, variant string) *BuildVariantTaskUnit
 	if bv == nil {
 		return nil
 	}
+	if bv.IsInDisplayTask(task) {
```
Comment (bsamek) [inline model/project.go]: Should this warn?

Comment (richardsamuels) [reply model/project.go]: Probably not. The point is that execution tasks don't exist from the user's perspective, so silently ending seems like the right call.

**Comment by john-m-liu** [comment]
Apparently Richard already added a good amount of test coverage for this. I changed the behavior for this and also fixed the test

File: model/project_test.go [hunk 2]
```diff
@@ -773,36 +773,6 @@ func (s *projectSuite) TestBuildProjectTVPairsWithAlias() {
 	}
 }
 
-func (s *projectSuite) TestBuildProjectTVPairsWithBadBuildVariant() {
```
Comment (richardsamuels) [inline model/project_test.go]: Why are you deleting this one?

File: model/project.go [hunk 3]
```diff
@@ -759,6 +759,17 @@ func (p *Project) FindTaskForVariant(task, variant string) *BuildVariantTaskUnit
 	return nil
 }
 
+func (bv *BuildVariant) InDisplayTask(taskName string) string {
```
Comment (bsamek) [inline model/project.go]: This seems like it should return a bool.

Comment (john-m-liu) [reply model/project.go]: It did at one point. I renamed it

**Pull request merged**